docs/user/monitor_experiments_with_tensorboard.md

```diff
@@ -0,0 +1,178 @@
+# Monitor Your Experiments with TensorBoard
+
+[TensorBoard](https://www.tensorflow.org/tensorboard) is a powerful tool that allows you to monitor and visualize an experiment. This section introduces how to set up TensorBoard logging in garage, and how to view/share your TensorBoard logs.
+
+## Prerequisite
+The TensorBoard package should be installed upon garage's installation. It can be also manually installed via:
+```bash
+$ pip install tensorboard
+```
+
+## Setup Your Experiment with `@wrap_experiment`
```

Comment:
i think you can just link out to the other documentation on how to use wrap_experiment. this doc can be very short, in other words